.eslintrc

```diff
@@ -54,13 +54,6 @@
     "function-paren-newline": "off",
     "generator-star-spacing": "off",
     "global-require": "off",
-    "indent": [
```

Comment:
Deleted this because eslint and prettier lint were clashing with the indentation rule. There would be a warning if eslint was broken, and a warning if prettier was broken, but you would end up with one or the other!